Update algorithm-secrets.md

lemonez opened the pull request:
clarify some wording, add copy-pasteable code sample, make concise

[DOCS-XXX](https://algorithmia.atlassian.net/jira/software/projects/DOCS/boards/16/backlog) - Brief description / title of ticket

Additional context/notes if appropriate (e.g., specifics about what's changing, links to epic or other PRs, etc.) 

## Checklist
- [ ] My PR title includes a relevant Jira ticket name
- [ ] If no associated Jira ticket, my PR title is informative (individual commit messages are squashed in this repo)
- [ ] Unnecessary text (Notes, TODOs, etc.) has been removed
- [ ] Screenshots, if present, are full-width and follow the [Screenshot guidelines](https://algorithmia.atlassian.net/wiki/spaces/CUSTOMERS/pages/1634861478/CFD+Style+Guide#Screenshots)
- [ ] Grammar, style, tone, punctuation, etc. follow the Algorithmia [manual of style](https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing)
- [ ] Headers are “Sentence case with Proper Nouns capitalized”
- [ ] Placeholder strings are UPPER_SNAKE_CASE and are named as in the [Glossary](https://docs.google.com/document/d/1bYs0j_a4v8LbkbS8r0x2SZ_J6mZ4sIt4w_60dq8iGh4/edit#heading=h.qov2yks2a685) where appropriate

lemonez committed:
Update algorithm-secrets.md

clarify some wording, add copy-pasteable code sample, make concise

lemonez commented:
pressed "create" before filling out template. this is for [DOCS-206](https://algorithmia.atlassian.net/browse/DOCS-206)

TODO: create (or find if it exists) admin-side secret store config docs

lemonez committed:
Update algorithm-secrets.md

fix quotes

lemonez committed:
add python highlighting

stevielkim commented on _pages/platform/algorithm-secrets.md:
```diff
@@ -29,34 +30,45 @@ This feature is available to [Algorithmia Enterprise](/enterprise) users only.
-If you're new to Algorithmia and would like to learn more about our product and using external secret stores, please [contact our sales team](https://info.algorithmia.com/contact-sales). We'd love to hear from you!
+Algorithmia Enterprise's secret store enables cluster admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
 
 ## Managing algorithm secrets
```
@williamhansen could you review this language since I believe that we don't want to refer to them as "algorithm secrets" but I'm not sure if "secret store" is the relevant term in this context

lemonez replied:
@StephanieKim can we just use "Secret Store" for now? can always change later, I'm hoping to get this PR through, I missed that it was still sitting here

stevielkim commented on _pages/platform/algorithm-secrets.md:
```diff
@@ -1,26 +1,27 @@
 ---
 
-Algorithm secrets let you decouple sensitive data like passwords, access tokens, and credentials from your algorithm source code. With this feature, you can securely store secrets on Algorithmia and access them inside your algorithms using environment variables.
+Algorithmia's secret store enables you to decouple sensitive data like passwords, access tokens, and credentials from your algorithm source code. With this feature, you can store your sensitive data securely on Algorithmia and access individual secret values from within your algorithms using environment variables.
```
```suggestion
Algorithmia's Secret Store enables you to decouple sensitive data like passwords, access tokens, and credentials from your algorithm source code. With this feature, you can store your sensitive data securely on Algorithmia and access individual secret values from within your algorithms using environment variables.
```

stevielkim commented on _pages/platform/algorithm-secrets.md:
```diff
@@ -29,34 +30,45 @@ This feature is available to [Algorithmia Enterprise](/enterprise) users only.
-Algorithmia Enterprise allows platform admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
-
-If you're new to Algorithmia and would like to learn more about our product and using external secret stores, please [contact our sales team](https://info.algorithmia.com/contact-sales). We'd love to hear from you!
+Algorithmia Enterprise's secret store enables cluster admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
```
```suggestion
Algorithmia Enterprise's Secret Store enables cluster admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
```

stevielkim commented on _pages/platform/algorithm-secrets.md:
```diff
@@ -29,34 +30,45 @@ This feature is available to [Algorithmia Enterprise](/enterprise) users only.
-Algorithmia Enterprise allows platform admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
-
-If you're new to Algorithmia and would like to learn more about our product and using external secret stores, please [contact our sales team](https://info.algorithmia.com/contact-sales). We'd love to hear from you!
+Algorithmia Enterprise's secret store enables cluster admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
```
@williamhansen any changes you want to make here?

williamhansen replied:
I want to be sure that it's clear that customers can integrate their Vault tool with Algorithmia's Secret Store.

lemonez replied:
changed wording to clarify.

lemonez committed:
Update _pages/platform/algorithm-secrets.md

Co-authored-by: StephanieKim <stephanie.kim76@gmail.com>

lemonez committed:
Update algorithm-secrets.md

capitalize Secret Store

lemonez committed:
Update algorithm-secrets.md

change wording to indicate more explicitly that Secret Store enables connection to external vaults

lemonez commented:
@StephanieKim I think my changes address William's comments. I would love to get this in before Tues deploy, as I'm working on the admin side of this and would love to have the updates live to work off of.
